### PR TITLE
UX: fix date time modal on mobile

### PR DESCRIPTION
**Pull request by chapoi**
Oversight regression due to modal changes

Before:
image

After
image

### CODE REVIEW

**Comment by CvX** [comment]
Can the calendar part be centered? (without too much work 😅)

**Comment by chapoi** [comment]
Oh it is, saw it after i put the img

File: app/assets/stylesheets/mobile/modal-overrides.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    flex-direction: column;
+    gap: 0.5em;
+  }
+  .calendar-date-time-input {
+    width: 100%;
+    margin: 0 0 1rem 0;
+  }
+
+  .picker-panel {
+    //mysteriously needed to prevent it from overlapping the timezone-input
```
Comment (CvX) [inline app/assets/stylesheets/mobile/modal-overrides.scss:12]: 👻😄

**Review by CvX — APPROVED** [review]

**Comment by chapoi** [comment]
@CvX the failing test isnt an issue?

**Pull request merged**